src/components/screens/home/balance-card.tsx

```diff
@@ -19,6 +21,10 @@ const BalanceCard = () => {
   const setError = useSetAtom(errorAtom);
   const [loading, setLoading] = useState(false);
   const [personTotalTime, setPersonTotalTime] = useState<PersonTotalTime>();
+  const currentDate = DateTime.now();
+  const beforeDate = currentDate.minus({ days: 1 }).startOf("day");
```

Comment:
That's true, didn't notice that. Would it be better to set for example as:

currentDate.minus({ days: 1 }).set({ hour: 23, minute: 59 });

Reply:
currentDate.startOf("day"); shows the current date we're living, but I suppose that's okay since it excludes the estimated balance of today which confuses the balance.